packages/components/src/styles/components/dialog-primitive.scss

```diff
@@ -89,6 +89,7 @@ $hds-dialog-primitive-vertical-padding: 16px;
 
 .hds-dialog-primitive__title {
   flex-grow: 1;
+  margin: 0;
```

Comment:
**Question:** this is the only line in this file I wanted to change.. but I guess my editor reformatted a bunch of stuff. Should I revert to only change this line?

Reply:
I see the same thing locally, but given these changes don't show up when running `yarn lint` it's possible I gave you the wrong editor settings for vscode 🤔 . I'm curious to see what others say/what their setting for this is:

```
"[scss]": {
        "editor.defaultFormatter": "esbenp.prettier-vscode",
        "editor.formatOnSave": true
    },
```

Reply:
> Should I revert to only change this line?

yes, please, until we fire it out. it looks like the linter couldn't find the config and used the defaults

Reply:
your setup looks correct, Brian

Reply:
👍🏽 when I removed that config it didn't make the changes, so maybe there is something I need to fix.